### ISSUE TITLE
Optimise XGBoost

### ISSUE THREAD

**Issue by mastoffel** [alan-turing-institute/autoemulate#100]
`XGBoost` is currently very slow and not better than gradient boosting. There are a few things to optimize:

* parallelisation: `XGBoost` can be parallelised for each model fit, but not sure how to make that work with parallelisation across model fits in the hyperparameter search.
* default parameters

**Comment by mastoffel** [comment]
Replaced XGBoost with LightGBM.
